src/Photos/Enums.cs

```diff
@@ -1,13 +1,20 @@
 using System;
+using System.Runtime.Versioning;
```

Comment:
This entire file should be reverted:

PHOTOS_API_SOURCES = \
	Photos/Enums.cs \